packages/@react-spectrum/tag/test/TagGroup.test.js

```diff
@@ -296,44 +317,74 @@ describe('TagGroup', function () {
 
     let tag = getByText('Tag 1');
     fireEvent.keyDown(tag, {key: props.keyPress});
+    expect(onRemoveSpy).toHaveBeenCalledTimes(1);
+    expect(onRemoveSpy).toHaveBeenCalledWith('1');
+  });
+
+  it('should remove tag when remove button is clicked', function () {
+    let {getAllByRole} = render(
+      <Provider theme={theme}>
+        <TagGroup aria-label="tag group" allowsRemoving onRemove={onRemoveSpy}>
+          <Item key="1" aria-label="Tag 1">Tag 1</Item>
+          <Item key="2" aria-label="Tag 2">Tag 2</Item>
+          <Item key="3" aria-label="Tag 3">Tag 3</Item>
+        </TagGroup>
+      </Provider>
+    );
+
+    let tags = getAllByRole('row');
+    fireEvent.click(tags[0]);
```

Comment:
best to use triggerPress
fireEvent will only fire the 'click' event, it won't do any of the pointer/mouse down